xlml/apis/gcp_config.py

```diff
@@ -27,13 +28,14 @@ class GCPConfig:
   Attributes:
     project_name: Name of a project to provision resource and run a test job.
     zone: The zone to provision resource and run a test job.
-    dataset_name: The option of dataset for metrics.
+    dataset_name: The option of dataset for metrics. If it is None, we do not
+      postprocess/store results.
     dataset_project: The name of a project that hosts the dataset.
     composer_project: The name of a project that hosts the composer env.
   """
 
   project_name: str
   zone: str
-  dataset_name: metric_config.DatasetOption
+  dataset_name: Optional[metric_config.DatasetOption] = None
```

Comment:
No, please don't set it as Optional. The system will always need to send metrics to BigQuery to monitor all tests status.

Reply:
I didn't even notice that.  Updated to force the dataset_name

Reply:
Do we still need the datasheet name even though we don't store any data? For example, the xlml test?

Reply:
Per offline discussion, we need the name.